ski_conditions/apps/app_scraping/tests.py

```diff
@@ -0,0 +1,3 @@
+# from django.test import TestCase
```

Comment:
Delete this file if you aren't going to write any tests now.

Reply:
Done. I've been working off of your scrape-refactor branch and have added to the tests there.